src/highlight.js

```diff
@@ -238,6 +241,42 @@ https://highlightjs.org/
     }
   }
 
+  function commonKeyword(word) {
```

Comment:
I would write the functions in the order in which they are called. Starting with "compileKeyword", then "scoreForKeyword", then "isCommonKeyword"

Reply:
I think I care less about this in JS and such a huge codebase (nothing seems organized)... but I reordered them.